ts/components/core/variables/IOColors.ts

```diff
@@ -185,132 +181,176 @@ export const IOColorsLegacy = {
+  "grey-150": IOColors["grey-100"],
+  "grey-50": IOColors["grey-50"],
+  white: IOColors.white
 });
 export type IOColorsNeutral = keyof typeof IOColorsNeutral;
 
-export const IOColorsNeutralDark: Record<
-  NonNullable<IOColorsNeutral>,
-  ColorValue
-> = {
-  white: IOColors.white,
-  grey50: IOColorsDark.grey50,
-  grey100: IOColorsDark.grey100,
-  grey200: IOColorsDark.grey200,
-  grey450: IOColorsDark.grey450,
-  grey650: IOColorsDark.grey650,
-  grey700: IOColorsDark.grey700,
-  grey850: IOColorsDark.grey850,
-  blackNew: IOColorsDark.blackNew
-};
-
-export type IOColorsNeutralDark = keyof typeof IOColorsNeutralDark;
-
 export const IOColorsTints = asIOColors({
-  blueNewDark: IOColors.blueNewDark,
-  blueNew600: IOColors.blueNew600,
-  blueNew: IOColors.blueNew,
-  blueNew200: IOColors.blueNew200,
-  blueNewLight: IOColors.blueNewLight,
-  blueNew100: IOColors.blueNew100,
-  blueNew50: IOColors.blueNew50,
-  turquoiseDark: IOColors.turquoiseDark,
-  turquoise: IOColors.turquoise,
-  turquoiseLight: IOColors.turquoiseLight,
-  turquoise100: IOColors.turquoise100,
-  turquoise50: IOColors.turquoise50
+  "blueIO-850": IOColors["blueIO-850"],
+  "blueIO-600": IOColors["blueIO-600"],
+  "blueIO-500": IOColors["blueIO-500"],
+  "blueIO-450": IOColors["blueIO-450"],
+  "blueIO-200": IOColors["blueIO-200"],
+  "blueIO-150": IOColors["blueIO-150"],
+  "blueIO-100": IOColors["blueIO-100"],
+  "turquoise-850": IOColors["turquoise-850"],
+  "turquoise-500": IOColors["turquoise-500"],
+  "turquoise-450": IOColors["turquoise-450"],
+  "turquoise-150": IOColors["turquoise-150"],
+  "turquoise-100": IOColors["turquoise-100"],
+  "turquoise-50": IOColors["turquoise-50"]
 });
 export type IOColorsTints = keyof typeof IOColorsTints;
 
-export const IOColorsTintsDark: Record<
-  NonNullable<IOColorsTints>,
-  ColorValue
-> = {
-  blueNewDark: IOColorsDark.blueNewDark,
-  blueNew600: IOColorsDark.blueNew600,
-  blueNew: IOColorsDark.blueNew,
-  blueNew200: IOColorsDark.blueNew200,
-  blueNewLight: IOColorsDark.blueNewLight,
-  blueNew100: IOColorsDark.blueNew100,
-  blueNew50: IOColorsDark.blueNew50,
-  turquoiseDark: IOColorsDark.turquoiseDark,
-  turquoise: IOColorsDark.turquoise,
-  turquoiseLight: IOColorsDark.turquoiseLight,
-  turquoise100: IOColorsDark.turquoise100,
-  turquoise50: IOColorsDark.turquoise50
-};
+// export const IOColorsTintsDark: Record<
+//   NonNullable<IOColorsTints>,
+//   ColorValue
+// > = {
+//   blueNewDark: IOColorsDark.blueNewDark,
+//   blueNew600: IOColorsDark.blueNew600,
+//   blueNew: IOColorsDark.blueNew,
+//   blueNew200: IOColorsDark.blueNew200,
+//   blueNewLight: IOColorsDark.blueNewLight,
+//   blueNew100: IOColorsDark.blueNew100,
+//   blueNew50: IOColorsDark.blueNew50,
+//   turquoiseDark: IOColorsDark.turquoiseDark,
+//   turquoise: IOColorsDark.turquoise,
+//   turquoiseLight: IOColorsDark.turquoiseLight,
+//   turquoise100: IOColorsDark.turquoise100,
+//   turquoise50: IOColorsDark.turquoise50
+// };
```

Comment:
Are these comments necessary?

Reply:
Oops. I should remember to check and clean everything before submitting the PR. My mistake. Now they are all gone.